common-js/Biconomy.js

```diff
@@ -845,7 +856,16 @@ function _sendSignedTransaction() {
             forwarderToUse = _context7.sent;
             //Update the verifyingContract in domain data
             forwarderDomainData.verifyingContract = forwarderToUse;
-            domainDataToUse = forwarderDomainDetails[forwarderToUse]; // Update the verifyingContract field of domain data based on the current request
+            domainDataToUse = forwarderDomainDetails[forwarderToUse];
+
+            if (customDomainName) {
```

Comment:
should a check for domainDataToUse be added as well ? can forwarderDomainDetails[forwarderToUse] be undefined ?

Reply:
It shouldn't be as it comes from systeminfo for supported forwarders. I have handled undefined once in the overall transaction flow which comes before this.